client/src/main/java/se/fortnox/reactivewizard/client/HttpClient.java

```diff
@@ -509,6 +519,16 @@ protected String getPath(Method method, Object[] arguments, JaxRsMeta meta) {
                     } else {
                         path = path.replaceAll("\\{" + ((PathParam)annotation).value() + "\\}", this.urlEncode(this.serialize(value)));
                     }
+                } else if (annotation instanceof BeanParam) {
+                    if (value == null) {
+                        continue;
+                    }
+                    beanParamCache
+                        .computeIfAbsent(types[i], this::getBeanParamGetters)
+                        .forEach(v -> {
+                            args.add(v.getValue1().apply(value));
```

Comment:
My first question when reading this line is " what type is Value1 and what type is value2 ?" Can we add them to local variables, just to make it easier to follow the code? Now when im reading the code, i need to go into the getBeanParamGetter method to see what it might return.

Reply:
That is indeed the downside of the Tuple... I will add local vars.